src/sidebar/templates/annotation.html

```diff
@@ -56,8 +56,8 @@
 
   <div class="annotation-body u-layout-row tags tags-read-only"
        ng-if="(vm.canCollapseBody || vm.state().tags.length) && !vm.editing()">
-    <ul class="tag-list" aria-label="Annotation tags">
-      <li class="tag-item" ng-repeat="tag in vm.state().tags">
+    <ul class="tag-list" role="list" aria-label="Annotation tags">
```

Comment:
Usually it isn't necessary to add a semantic role if it is already implied by the HTML element (ie. `ul` and `li` imply a list and list item don't they). Did you find that this made a practical difference?

Reply:
@robertknight, this was new to me too. I found that, at least with ChromeVox, it would not read out the aria-label on the ul or li tags until I defined the roles. Since they are only reducent at worse, I'm included to leave them so there is no chance of the screen readers missing the label.

It does look like an open point of confusion: https://github.com/w3c/aria/issues/756

Reply:
I wonder if this is a semantic grey area because, while `ul`s are semantically assumed to contain a collection of list items, it is not part of the semantic structure of a `ul` element itself to provide a heading or description for what it contains. That is, I wonder if it is presumed that one should add a heading or other appropriate semantic element to the document structure. Or it could be a questionable implementation choice on the part of the screenreader(s).

In any case, I'm wary of adding any ARIA roles other than [landmark roles](https://www.washington.edu/accessibility/web/landmarks/), especially to "fix" the behavior of screenreaders (trying to munge document structure specifically to satisfy screenreader implementations is a depressing, and ultimately un-winnable, arms race in my opinion). I fear that this might be kicking the can down the road and could lead to some tech debt here. In case it's not abundantly clear, I have a strong "do as little as possible" philosophy when it comes to trying to control browsers and a11y tools—I tend to find in the long run it behooves one to stick as close to the bone/standards as is feasible and let the browsers (and screenreaders) do their jobs based on the semantic document structure. In other words, feel free to set aside my opinion as I am but one voice in the peanut gallery!

I wonder if this tension here means we might consider reframing the problem. Is it only users of screenreaders who might find the list of tags ambiguous? Is it obvious to all other users that these are tags? That is, is the ambiguity posed by screenreaders not reading out the `aria-label` significantly different from the ambiguity all users face when looking at that list? Perhaps so, perhaps to sighted users the affordances given by the way the tags are presented fills in for some of that ambiguity.

In any case, tl;dr, I vote not adding ARIA `role` attributes to these elements.